tensorflow/lite/micro/kernels/micro_tensor_utils.h

```diff
@@ -0,0 +1,867 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
```

Comment:
nit: WDYT: add a comment here that indicates "This file is branched from tensorflow/lite.../tensor_utils.h. TFLM needs to use its own b/c ..."

Reply:
Added detailed explanation why it's needed.